config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
+  get 'users/index'
+  get 'posts/index'
   root 'pages#hello'
```

Comment:
- [ ] Instead of writing routes for all separately, you could write like
 ```
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```